### PR TITLE
[fix](function) support some ip functions support vector and constant calculations together

### PR DESCRIPTION
**Pull request by sjyango**
## Proposed changes

Issue Number: close #xxx

- fix ip functions: `is_ip_address_in_range`, `ipv4_cidr_to_range`, `ipv6_cidr_to_range`.

- before this pr:
when calculating vector and constant together, these functions will core dump.

- after this pr:
fix core dump bugs.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sjyango** [comment]
run buildall

**Comment by sjyango** [comment]
run external

**Review by zclllyybb — COMMENTED** [review]
review after finishing all const optimization

**Comment by sjyango** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]
LGTM for const dealing

File: be/src/vec/functions/function_ip.h [hunk 1]
```diff
@@ -714,24 +718,83 @@ class FunctionIsIPAddressInRange : public IFunction {
...
+        return Status::OK();
+    }
+
+    static void vector_vector(const ColumnString* str_addr_column, const NullMap* null_map_addr,
+                              const ColumnString* str_cidr_column, const NullMap* null_map_cidr,
+                              ColumnUInt8::Container& col_res_data, size_t input_rows_count) {
+        DCHECK(str_addr_column->size() == input_rows_count);
+        DCHECK(str_cidr_column->size() == input_rows_count);
+        for (size_t i = 0; i < input_rows_count; ++i) {
+            if (null_map_addr && (*null_map_addr)[i]) {
```
Comment (HappenLee) [inline be/src/vec/functions/function_ip.h:752]: auto idx = index_check_const(i, is_const_args[j])；

**Pull request closed without merging**